app/views/compute_resources_vms/form/vmware/_volume.html.erb

```diff
@@ -24,3 +24,4 @@
            "true",
            "false" %>
 <%= select_f f, :mode, compute_resource.disk_mode_types, :first, :last, { }, :class => "span5", :label => _("Disk mode"), :label_size => "col-md-2", :disabled => !new_host %>
+<%= select_f f, :controller_key, compute_resource.controller_ids, :first, :last, { }, :class => "span5", :label => _("SCSI Controller ID"), :label_size => "col-md-2" , :disabled => !new_host %>
```

Comment:
In the current github develop branch there are 2 lines of code which are using "new_host" instead of "new_vm". It's the vmware name and the disk mode. Should I open a new redmine issue and fix both?